registry/storage-drivers/s3.md

```diff
@@ -9,186 +9,29 @@ Amazon S3 or S3 compatible services for object storage.
+| `accesskey` | no     | Your AWS Access Key. If you use [IAM roles](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), omit to fetch temporary credentials from IAM.|
+| `secretkey`  | no   |Your AWS Secret Key. If you use [IAM roles](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), omit to fetch temporary credentials from IAM.|
+| `region`  |  yes  | The AWS region in which your bucket exists. For the moment, the Go AWS library in use does not use the newer DNS based bucket routing.|
+| `regionendpoint`  |  no  | Endpoint for S3 compatible storage services (Minio, etc).|
+| `bucket`  | yes   | The bucket name in which you want to store the registry's data.|
+| `encrypt`  | no   | Specifies whether the registry stores the image in encrypted format or not. A boolean value. The default is `false`.|
+| `keyid`  |  no  | Optional KMS key ID to use for encryption (encrypt must be true, or this parameter is ignored). The default is `none`.|
+| `secure`  |  no  | Indicates whether to use HTTPS instead of HTTP. A boolean value. The default is `true`.|
+| `skipverify`  | no   | Skips TLS verification when the value is set to `true`. The default is `false`.|
+| `v4auth`  |  no  | Indicates whether the registry uses Version 4 of AWS's authentication. The default is `true`.|
+| `chunksize`  |  no  | The S3 API requires multipart upload chunks to be at least 5MB. This value should be a number that is larger than 5 * 1024 * 1024.|
+| `rootdirectory`  | no  | This is a prefix that is applied to all S3 keys to allow you to segment data in your bucket if necessary. |
+| `storageclass`  |  no  | The S3 storage class applied to each registry file. The default is `STANDARD`.|
+| `s3accelerate`  |  no  | Specifies whether the registry should use S3 Transfer Acceleration. You must enable acceleration endpoint on a bucket before using this option. A boolean value. The default is `false`.|
 
 > **Note** You can provide empty strings for your access and secret keys to run the driver
 > on an ec2 instance and handles authentication with the instance's credentials. If you
 > use [IAM roles](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html),
 > omit these keys to fetch temporary credentials from IAM.
 
-`region`: The name of the aws region in which you would like to store objects (for example `us-east-1`). For a list of regions, you can look at http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
+`region`: The name of the aws region in which you would like to store objects (for example `us-east-1`). For a list of regions, see [Regions, Availability Zones, and Local Zones](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html).
```

Comment:
Looks like these descriptions overlap with the ones in the table above.

Don't think it's worth spending time on right now (as it would require checking the difference between both), but something I just noticed

Reply:
Sure, thanks.